Project2015To2017Tests/XamlTransformationTest.cs

```diff
@@ -90,23 +90,6 @@ public async Task TransformsPresentationPages()
 
 			Assert.AreEqual(3, project.AdditionalPropertyGroups.Count);
 
-			Assert.IsNotNull(project.AdditionalPropertyGroups[0].Attribute("Condition"));
```

Comment:
Does not relate to testing XAML transformation, moved to a new test for PropertySimplificationTransformation.